src/uu/tr/src/operation.rs

```diff
@@ -277,13 +278,19 @@ impl Sequence {
             separated_pair(Self::parse_backslash_or_char, tag("*"), digit1),
             tag("]"),
         )(input)
-        .map(|(l, (c, str))| {
+        .map(|(l, (c, cnt_str))| {
+            let result: Result<usize, ParseIntError>;
+            if cnt_str.chars().next().unwrap() == '0' {
+                result = usize::from_str_radix(cnt_str, 8);
+            } else {
+                result = usize::from_str_radix(cnt_str, 10);
+            };
```

Comment:
```suggestion
            let result = cnt_str.starts_with('0') {
                usize::from_str_radix(cnt_str, 8);
            } else {
                usize::from_str_radix(cnt_str, 10);
            };
```

Reply:
Thanks! You can add those "words" to the `// spell-checker:ignore` comments in that file.

Reply:
This PR has way too many commits than I wanted. Will you be able to squash them while merging or should I try to squash them and send another PR?

Also, please review and let me know if this looks okay.